scripts/qa_crowbarsetup.sh

```diff
@@ -3260,6 +3260,8 @@ function addfloatingip
     nova floating-ip-create | tee floating-ip-create.out
     floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
     nova add-floating-ip "$instanceid" "$floatingip"
+    # flush arp tables in case this floating ip was recently used for something else
+    arp -d "$floatingip"
```

Comment:
@rossella yep, I'll get an environment ready. I have temporarily disabled l2pop which seemed to have helped quite a bit.